src/pybind/mgr/cephadm/templates/services/ingress/haproxy.cfg.j2

```diff
@@ -85,6 +85,6 @@ backend backend
     default-server {{ default_server_opts|join(" ") }}
 {% endif %}
     {% for server in servers %}
-    server {{ server.name }} {{ server.ip }}:{{ server.port }}
+    server {{ server.name }} {{ server.ip }}:{{ server.port }} check weight 100
```

Comment:
There are 3 unit tests
```
FAILED cephadm/tests/test_services.py::TestIngressService::test_ingress_config_nfs_multiple_nfs_same_rank[False]
FAILED cephadm/tests/test_services.py::TestIngressService::test_ingress_config_nfs_multiple_nfs_same_rank[True]
FAILED cephadm/tests/test_services.py::TestIngressService::test_ingress_config_nfs_proxy_protocol
```
that generate haproxy configs that will have to be updated for make check to pass with this change.

Reply:
Thanks. Fixed the tests!